src/entry_node.cpp

```diff
@@ -29,16 +29,19 @@
 namespace ovms {
 
 Status EntryNode::execute(session_key_t sessionId, PipelineEventQueue& notifyEndQueue) {
-    // TODO this should be created in EntryNode::SetInputs, or special method for entry node called
-    // in event loop can be done at the end of part 3 or in future release
+    // this should be created in EntryNode::SetInputs, or special method for entry node called
```

Comment:
Since you only remove _TODO_ word I assume we have jira ticket for this?

Reply:
Yes - this is dynamic batch pipeline.